### DISCUSSION TITLE
Can i run native php on Windows ?

### DISCUSSION THREAD

**Discussion by Metaveos99** [NativePHP/laravel#205] [Q&A] [answered]
i have installed native php but when I'm trying to start the server using php artisan native:serve im getting this error:-

Uncaught Exception:

Error: spawn E:\Test\demo\vendor\nativephp\electroni resources\js\resources\php\php
ENOENT

at ChildProcess._handle.onexit (nodeiinteral/child_process:283:19)
at onErmorNT (nodeintemnal/child_process:476:16)

at process.processTicksAndRejections (nodeiinternal/process/task_queues:8221)

**Comment by simonhamp** [comment]
No. Windows is currently not supported.

**Comment by simonhamp** [comment, marked as answer]
[You can now!](https://github.com/orgs/NativePHP/discussions/278)
